src/couch/rebar.config.script

```diff
@@ -118,6 +118,11 @@ end,
             "-DXP_UNIX -I/usr/include/mozjs-60 -I/usr/local/include/mozjs-60 -std=c++14",
             "-L/usr/local/lib -lmozjs-60 -lm -std=c++14 -lc++"
         };
+    {unix, linux} when SMVsn == "60" ->
```

Comment:
@jiangphcn thanks for this, what happens if we just add `-lstdc++` to the `{unix, _}` case? Does it break other Linux builds? Or, is there a reason the `-std=c++14` line won't work on RHEL?

Reply:
@wohali thanks for your question and suggestion. I tested with Centos and Debian,  and it will not break if we just add -lstdc++ to the {unix, _} case. I change accordingly. Could you please check again?